crates/storage/db/src/abstraction/table.rs

```diff
@@ -69,6 +69,9 @@ pub trait Table: Send + Sync + Debug + 'static {
     type Value: Value;
 }
 
+/// Helper type containing key value pair.s
+pub type KeyValue<T> = (<T as Table>::Key, <T as Table>::Value);
```

Comment:
Where is this used? And if it is used somewhere, it should be in https://github.com/paradigmxyz/reth/blob/99a821aa78c03f4397a0d0f35ee55eb9e7f8b465/crates/storage/db/src/abstraction/common.rs#L2 like the other typedefs not directly related to tables

Reply:
It is used as a return type in a few function, we see to move it and reuse it in few more places.